sql/core/src/test/scala/org/apache/spark/sql/SQLQuerySuite.scala

```diff
@@ -3317,6 +3317,19 @@ class SQLQuerySuite extends QueryTest with SharedSparkSession {
         """.stripMargin).collect()
     }
   }
+
+  test("SPARK-29595: Unorder insertion in NAMED_STRUCT should throw exception") {
```

Comment:
I think this is not a bug, so can you drop the prefix `SPARK-29595`?

Reply:
Fixed.